CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 ## Release Date: UNRELEASED Valhalla 3.0
 
+## Release Date: 2018-07-24 Valhalla 3.0.0-rc.3
```

Comment:
Should probably note the change to async invocation here too.

Reply:
yep, this was a carry over from my other PR which I ultimately closed. Updating this now